Classes/Providers/OVP/OVPMediaProvider.swift

```diff
@@ -213,12 +221,23 @@ public class OVPMediaProvider: MediaEntryProvider {
                         mediaSources.append(mediaSource)
                     })
                     
+                    var metaDataItems: [(name: String, value: String)] = [(name: String, value: String)]()
```

Comment:
You don't need the extra type declaration. This will also work:
`var metaDataItems = [(name: String, value: String)]()`